addon-test-support/-private/mock-server.js

```diff
@@ -25,27 +25,27 @@ let createMock = async function(path, method, statusCode, response) {
 }
 
 export let mockServer = {
-  async get(path, response, status = 200) {
```

Comment:
without `await` in the method `async` keyword does make any difference

Reply:
I had similar issue from ember 3.16,  and I think it could be rewrite from

```javascript
async get(path, response, status = 200) {
```

to

```javascript
get: async function(path, response, status = 200) {
```

See: https://github.com/screwdriver-cd/ui/pull/576#issuecomment-668887311

Reply:
@adong yeah, that's an option - however async is just not needed here as there is no `await` in the method body